ballerina/field_types.bal

```diff
@@ -30,3 +30,25 @@ public type SheetNumericType int|decimal|boolean|float;
 # Generic type that can used to store any of basic types supported by sheets
 #
 public type SheetBasicType int|string|decimal|boolean|float;
+
+# Record type to represent error response coming from appscript
+#
+# + done - Status of the response
+# + response - Record to store the response if the request is successful
+# + error - Record to store the error if the request is failed
+public type AppScriptJsonResponseRecord record {
+    boolean done;
+    json response?;
+    ErrorRecord 'error?;
+};
+public type ErrorRecord record {
```

Comment:
Shall we add API description for these records

Reply:
Updated in [afd480a](https://github.com/ballerina-platform/module-ballerinax-persist.googlesheets/pull/40/commits/afd480aa5bbc7bf276898d8223c0e1885bd8d34c)